src/main/java/net/dv8tion/jda/internal/entities/ReceivedMessage.java

```diff
@@ -820,6 +821,22 @@ public MessageAction editMessageEmbeds(@Nonnull Collection<? extends MessageEmbe
         return ((MessageActionImpl) channel.editMessageEmbedsById(getId(), embeds)).withHook(interactionHook);
     }
 
+    @Nonnull
+    @Override
+    public MessageAction editComponents(@Nonnull Collection<? extends ComponentLayout> actionRows)
```

Comment:
This should be overridden in `SystemMessage` as unsupported as well, you can't edit system messages.